rest/route/host.go

```diff
@@ -400,6 +400,13 @@ func (ch *offboardUserHandler) Run(ctx context.Context) gimlet.Responder {
 	opts := model.APIHostParams{
 		UserSpawned: true,
 	}
+	if !ch.dryRun {
+		grip.Info(message.Fields{
+			"message": "executing user offboarding",
+			"context": "user offboarding",
```

Comment:
Context here just seems to be a repeat of message so I'm not sure that's useful. It might be worth logging this at 438 instead so we can also log the `toTerminate` response.